tests-e2e/features/assets/digital-twins/digital-twin-detail-permissions.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Digital Twin - Detail View for missing permissions
+
+  Background:
+    Given I log in to evan.network using vue
+
+  Scenario: Access foreign twin without permissions
+    When I am on the path "#/dashboard.vue.evan/detail.digital-twin.evan/0x56fc9d62fce20193cd0c1d597aa7919630895b0b"
+      Then I want to see a text including "Test Car Twin"
+      And I want to see a text including "Information"
```

Comment:
I think there es a sub-menu selector or something like this to target twin containers

Reply:
There is, but this is a sub sub menu and I didnt want to write yet another function just for it 😅